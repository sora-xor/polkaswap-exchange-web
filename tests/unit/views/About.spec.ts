// import { shallowMount, createLocalVue } from '@vue/test-utils'

// import About from '@/views/About.vue'
// import { SoramitsuElementsImport, TranslationMock } from '../../utils'

// const localVue = createLocalVue()
// SoramitsuElementsImport(localVue)

// describe('About.vue', () => {
//   beforeEach(() => {
//     TranslationMock(About)
//   })

//   it('should renders correctly', () => {
//     const wrapper = shallowMount(About, { localVue })
//     expect(wrapper.element).toMatchSnapshot()
//   })
// })

describe('About test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
