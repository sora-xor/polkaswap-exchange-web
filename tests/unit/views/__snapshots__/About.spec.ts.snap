// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue should renders correctly 1`] = `
<div
  class="layout s-flex"
>
  <div
    class="banner s-flex"
  />
   
  <div
    class="content"
  >
    <div
      class="terms s-flex"
    >
      <span>
        
      </span>
       
      <div
        class="web3-logo s-flex"
      >
        <span>
          
        </span>
         
        <i
          class="logo"
        />
      </div>
    </div>
     
    <s-divider-stub
      contentposition="center"
      direction="horizontal"
      type="secondary"
    />
     
    <div
      class="links s-flex"
    >
      <s-button-stub
        borderradius="small"
        icon=""
        nativetype="button"
        size="medium"
        tabindex="0"
        tooltip=""
        tooltipplacement="top"
        type="primary"
      >
        
        
      
      </s-button-stub>
       
      <s-button-stub
        borderradius="small"
        icon="external-link"
        nativetype="button"
        size="medium"
        tabindex="0"
        tooltip=""
        tooltipplacement="top"
        type="link"
      >
        
        
      
      </s-button-stub>
       
      <s-button-stub
        borderradius="small"
        icon="external-link"
        nativetype="button"
        size="medium"
        tabindex="0"
        tooltip=""
        tooltipplacement="top"
        type="link"
      >
        
        
      
      </s-button-stub>
    </div>
     
    <s-divider-stub
      contentposition="center"
      direction="horizontal"
      type="secondary"
    />
     
    <div
      class="articles s-flex"
    >
      <s-row-stub
        align="top"
        gutter="0"
        justify="start"
      >
        <s-col-stub
          lg="3"
          md="4"
          offset="0"
          pull="0"
          push="0"
          sm="6"
          span="12"
          xs="12"
        >
          <s-card-stub
            borderradius="small"
            class="article-card"
            clickable="true"
            shadow="hover"
          >
             
            <span>
              
            </span>
          </s-card-stub>
        </s-col-stub>
        <s-col-stub
          lg="3"
          md="4"
          offset="0"
          pull="0"
          push="0"
          sm="6"
          span="12"
          xs="12"
        >
          <s-card-stub
            borderradius="small"
            class="article-card"
            clickable="true"
            shadow="hover"
          >
             
            <span>
              
            </span>
          </s-card-stub>
        </s-col-stub>
        <s-col-stub
          lg="3"
          md="4"
          offset="0"
          pull="0"
          push="0"
          sm="6"
          span="12"
          xs="12"
        >
          <s-card-stub
            borderradius="small"
            class="article-card"
            clickable="true"
            shadow="hover"
          >
             
            <span>
              
            </span>
          </s-card-stub>
        </s-col-stub>
        <s-col-stub
          lg="3"
          md="4"
          offset="0"
          pull="0"
          push="0"
          sm="6"
          span="12"
          xs="12"
        >
          <s-card-stub
            borderradius="small"
            class="article-card"
            clickable="true"
            shadow="hover"
          >
             
            <span>
              
            </span>
          </s-card-stub>
        </s-col-stub>
      </s-row-stub>
    </div>
  </div>
</div>
`;
